### ISSUE TITLE
[Bug]: check_dns does not work because nslookup is provided by BusyBox

### ISSUE THREAD

**Issue by dg76** [manios/docker-nagios#52]
### What happened?

When using `manios/nagios:latest` and then using `check_dns` it always results in this error:

```
DNS CRITICAL - '/usr/bin/nslookup -sil' msg parsing exited with no address 
```

When trying to run `nslookup -sil` manually this is the result:

```
BusyBox v1.31.1 () multi-call binary.

Usage: nslookup [-type=QUERY_TYPE] [-debug] HOST [DNS_SERVER]

Query DNS about HOST

QUERY_TYPE: soa,ns,a,aaaa,cname,mx,txt,ptr,any
```

Running `apk add bind-tools` fixes the error. Afterward running `nslookup -sil` returns this result:

```
nslookup -sil wiki.linuxquestions.org
Server:		127.0.0.11
Address:	127.0.0.11#53

Non-authoritative answer:
Name:	wiki.linuxquestions.org
Address: 104.237.137.190
```

And `check_dns` works correctly afterward, too. So I would suggest to incude the "bind-tools" package to ensure that `check_dns` can be used.

Thank you very much for creating and maintaining this image! :-)

### Image information

* build: 8
* gitCommit: 69c1a3a190b865ecaf765c50c5b2dcb1042bb9e8
* homepage: https://www.nagios.com/
* maintainer: Christos Manios <maniopaido@gmail.com>
* nagiosPluginsVersion: 2.3.3
* nagiosVersion: 4.4.6
* name: Nagios
* nrpeVersion: 4.0.3

### Image architecture

x86

**Comment by manios** [comment]
Ho @dg76 !
Thank you for your issue. The issue that you mentioned has already been resolved (#37) Can you please upgrade to the `Build 9` of the image and let me know if it worked for you? 

```bash
docker pull manios/nagios:latest
# OR
docker pull manios/nagios:build-9
```
Thanks,
Christos

**Comment by dg76** [comment]
Hi @manios,

I am sorry, you are right! Now it works fine with "latest". It is strange because I had already used "latest" when the error occurred. I don't know what happened, maybe it used a cached, older version of "latest"? 🤔 Anyway now it shows "build: 9" when using "latest" and it works fine. Thank you very much! 😊
